sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1208,6 +1208,14 @@ class Lasso(ElasticNet):
     If so, then additionally check whether the dual gap is smaller than `tol` times
     :math:`||y||_2^2 / n_{\text{samples}}`.
```

Comment:
There is also a bad rendering of `\text{n_samples}` here. We should fix it at the same time.

Reply:
I fixed it by adding a backslash to `\\text`.